### ISSUE TITLE
symbol lookup error: undefined symbol: FT_Get_Color_Glyph_Paint

### ISSUE THREAD

**Issue by gww-parity** [vector-im/element-web#19015]
### Steps to reproduce

```
$ element-desktop 
/usr/lib/electron13/electron: symbol lookup error: /usr/lib/electron13/electron: undefined symbol: FT_Get_Color_Glyph_Paint
```

more debugging:

```
$ LD_DEBUG=symbols element-desktop
<... cut ...>
   1893627:     symbol=FT_Get_Color_Glyph_Paint;  lookup in file=/usr/lib/libkeyutils.so.1 [0]
   1893627:     symbol=FT_Get_Color_Glyph_Paint;  lookup in file=/usr/lib/libgpg-error.so.0 [0]
   1893627:     symbol=FT_Get_Color_Glyph_Paint;  lookup in file=/usr/lib/libgomp.so.1 [0]
   1893627:     /usr/lib/electron13/electron: error: symbol lookup error: undefined symbol: FT_Get_Color_Glyph_Paint (fatal)
/usr/lib/electron13/electron: symbol lookup error: /usr/lib/electron13/electron: undefined symbol: FT_Get_Color_Glyph_Paint
```

Distro: Arch

Packages:

https://archlinux.org/packages/community/x86_64/element-desktop/

```
$ pacman -Qi element-desktop electron13
Name            : element-desktop
Version         : 1.7.34-2
Description     : Glossy Matrix collaboration client — desktop version.
Architecture    : x86_64
URL             : https://element.io
Licenses        : Apache
Groups          : None
Provides        : None
Depends On      : element-web=1.7.34  electron13  sqlcipher
Optional Deps   : None
Required By     : None
Optional For    : None
Conflicts With  : None
Replaces        : riot-desktop
Installed Size  : 38.70 MiB
Packager        : Nicola Squartini <tensor5@gmail.com>
Build Date      : 2021-09-07T03:00:52 CEST
Install Date    : 2021-09-13T18:07:49 CEST
Install Reason  : Explicitly installed
Install Script  : No
Validated By    : Signature

Name            : electron13
Version         : 13.3.0-1
Description     : Build cross platform desktop apps with web technologies
Architecture    : x86_64
URL             : https://electronjs.org/
Licenses        : MIT  custom
Groups          : None
Provides        : None
Depends On      : c-ares  ffmpeg  gtk3  libevent  libnghttp2  libxslt  minizip  nss  re2  snappy
Optional Deps   : kde-cli-tools: file deletion support (kioclient5) [installed]
                  libappindicator-gtk3: StatusNotifierItem support [installed]
                  pipewire: WebRTC desktop sharing under Wayland [installed]
                  trash-cli: file deletion support (trash-put) [installed]
                  xdg-utils: open URLs with desktop's default (xdg-email, xdg-open) [installed]
Required By     : element-desktop
Optional For    : None
Conflicts With  : None
Replaces        : None
Installed Size  : 169.02 MiB
Packager        : Nicola Squartini <tensor5@gmail.com>
Build Date      : 2021-09-02T02:52:55 CEST
Install Date    : 2021-09-13T18:07:49 CEST
Install Reason  : Installed as a dependency for another package
Install Script  : No
Validated By    : Signature
```

### What did you expect?

Application to run

### What happened?

Crashed on start as in logs above.

### Operating system

Arch Linux

### Application version

1.7.34-2

### How did you install the app?

pacman -Syy element-desktop

### Homeserver

N/A

### Have you submitted a rageshake?

No

**Comment by gwpl** [comment]
reported here: https://bugs.archlinux.org/task/72122

**Comment by gwpl** [comment]
it's critical to upgrade due to https://matrix.org/blog/2021/09/13/vulnerability-disclosure-key-sharing
so I consider making smooth upgrade path important for fixing this bug in "production" (i.e. on users machines).

**Comment by kittykat** [comment]
Hi, thank you for reporting this issue. I don't think this is a problem in element-desktop: I'm on Arch and the latest versions available are 1.8.4-2 (and electron 13.4.0-1). These work for me and I cannot reproduce the problem that you are seeing, so I will close this issue for now. Please update to the latest version and re-check that if the issue persists. If the update fails, you may want to try uninstalling and re-installing element-desktop.

If the problem is indeed in the latest element-desktop, please comment with @kittykat and I will re-open this issue for you.
